private_sb/git_changes.txt

```diff
@@ -0,0 +1,1176 @@
+Microsoft Windows [Version 10.0.18363.1139]
```

Comment:
Looks like all the files in this folder can be removed for PR

Reply:
removed it for the PR (branched it out though)